### PR TITLE
Changed empty string to default value

### PR DESCRIPTION
**Pull request by hemantmm**
Fixes github.com/https://github.com/vector-im/element-web/issues/23458
Updated the logic as suggested please check;
changed the empty string to RoomEvent.name, please guide me if something is not appropriate. Thank you.

               this.state.readMarkerEventId ?? "",  // changed to below line

               this.state.readMarkerEventId ?? RoomEvent.Name,

---
This PR currently has none of the required changelog labels.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

### CODE REVIEW

File: src/components/structures/TimelinePanel.tsx [hunk 1]
```diff
@@ -59,6 +59,10 @@ import { getKeyBindingsManager } from "../../KeyBindingsManager";
 import { KeyBindingAction } from "../../accessibility/KeyboardShortcuts";
 import { haveRendererForEvent } from "../../events/EventTileFactory";
 
+
+// @ts-ignore
```
Comment (weeman1337) [inline src/components/structures/TimelinePanel.tsx:63]: There should be no `@ts-ignore` in new code.

File: src/components/structures/TimelinePanel.tsx [hunk 2]
```diff
@@ -1058,7 +1062,11 @@ class TimelinePanel extends React.Component<IProps, IState> {
             } else {
                 cli.setRoomReadMarkers(
                     roomId,
-                    this.state.readMarkerEventId ?? "",
+                    // this.state.readMarkerEventId ?? "",
+                    // this.state.readMarkerEventId ?? JoinedRoomEvent(cli)[0],
+                    this.state.readMarkerEventId ?? RoomEvent.Name,
```
Comment (weeman1337) [inline src/components/structures/TimelinePanel.tsx:1067]: I don't think it is correct to use the room event name as fallback here.

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Hey @hemantmm thank you for your contribution.

Please have a look at my review comments.
Since this is a bugfix, can you provide a test case as well?

**Comment by t3chguy** [comment]
Closing as abandoned.

**Pull request closed without merging**